docs/proposals/20200329-join.md

```diff
@@ -0,0 +1,48 @@
+# KubeOne Join
+
+**Author:** Mohamed Elsayed ([@moelsayed](https://github.com/moelsayed))  
+**Status:** **Draft** | Review | Final
+**Created:** 2020-03-29
+**Last updated:** 2020-03-29
+
+## Abstract
+KubeOne initially supported deploying control plane nodes only, with the option to deploy machine-controller and dynamically provision worker nodes on a configured cloud provider.
+
+PR [#834](https://github.com/kubermatic/kubeone/pull/834) adds support for configuring static worker nodes to be deployed during initial cluster provisioning.
+
+The `join` command should provide allow users to easily add additional worker nodes to an existing KubeOne cluster.
+
+Currently, this is only doable by applying several manual steps on each worker node to deploy the required packages/files and execute `kubeadm join`.
+
+This new subcommand should allow users to add new static worker nodes to the clusters with a single command, and include these nodes in upgrade process as well.
+
+## Goals
+* Implement the `join` subcommand.
```

Comment:
To my understanding, the plan was that we don't want to have the `join` command, but instead to extend the `install` command (like in #834). Do we actually want the `join` command? Has the plan changed or I misunderstood something?